deploy/charts/emqx/templates/StatefulSet.yaml

```diff
@@ -120,8 +120,12 @@ spec:
             containerPort: {{ .Values.emqxConfig.EMQX_LISTENER__WSS__EXTERNAL | default 8084 }}
           - name: dashboard
             containerPort: {{ .Values.emqxConfig.EMQX_DASHBOARD__LISTENER__HTTP | default 18083 }}
-          {{- if not (empty .Values.emqxConfig.EMQX_DASHBOARD__LISTENER__HTTPS) }}
+          {{- if not (empty .Values.emqxConfig.EMQX_LISTENER__TCP__INTERNAL) }}
           - name: dashboardtls
+            containerPort: {{ .Values.emqxConfig.EMQX_LISTENER__TCP__INTERNAL }}
+          {{- end }}
+          {{- if not (empty .Values.emqxConfig.EMQX_DASHBOARD__LISTENER__HTTPS) }}
+          - name: internalmqtt
```

Comment:
It looks like line 124 should be `- name: internalmqtt` and line 128 should be `- name: dashboardtls`